src/FSLibrary/MissionMixedImageLoadGeneration.fs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionMixedImageLoadGeneration
+
+open stellar_dotnet_sdk
+open StellarCoreSet
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarCoreHTTP
+
+let mixedImageLoadGeneration (context: MissionContext) =
+    let oldNodeCount = context.oldImageNodeCount
+    let newNodeCount = 3 - context.oldImageNodeCount
+    let oldKeys = Array.init oldNodeCount (fun _ -> KeyPair.Random())
```

Comment:
I'm new to F#, but it seemed like I can't skip field when initializing a type. Also I don't really see the code that would populate keys in case if I provided an empty array. I would appreciate some example to look at in case if I'm missing something.

Reply:
Sorry I wasn't super clear - you can use `MakeLiveCoreSet` which populates keys based on node count. For example, like we do here: https://github.com/stellar/supercluster/blob/65c2e47104a76417f141b92f6aaf706cd1d1455a/src/FSLibrary/MissionComplexTopology.fs#L17

Reply:
Thanks for the pointer, updated the code to use this.